src/transformers/reverse_path.ts

```diff
@@ -0,0 +1,119 @@
+import { SVGPathData } from '../index.js';
+import { SVGPathDataTransformer } from '../SVGPathDataTransformer.js';
+import type { SVGCommand } from '../types.js';
+
+/**
+ * Reverses the order of path commands to go from end to start
+ * IMPORTANT: This function expects absolute commands as input.
+ * It doesn't convert relative to absolute - use SVGPathDataTransformer.TO_ABS() first if needed.
+ * @param commands SVG path commands in absolute form to reverse
+ * @returns New SVG commands in reverse order with absolute coordinates
+ */
+export function REVERSE_PATH(commands: SVGCommand[]): SVGCommand[] {
```

Comment:
i'm not sure if this command should take `SVGCommand[]` or `SVGPathData` as argument